pages/fellows/SantamRC.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: fellow
+pagetype: fellow
+shortname: SantamRC
+permalink: /fellows/SantamRC.html
+fellow-name: Santam Roy Choudhury
+title: Santam Roy Choudhury - IRIS-HEP Fellow
+active: true
+dates:
+  start: 2021-06-01
+  end: 2021-08-31
+photo: /assets/images/team/Santam-Roy-Choudhury.jpg
+institution: National Institute of Technology, Durgapur
+e-mail: santamdev404@gmail.com
+project_title: Automating Awkward Array Testing
+project_goal: >
+    Awkward Array is a popular library for nested, variable-sized data, including
+    arbitrary-length lists, records, mixed types, and missing data, using NumPy-like idioms.
+    The project involves automating tests for various “Kernel Functions” in the library by working
+    on selecting appropriate input values for test cases of the kernel functions.
+    Here Hypothesis Testing Tool would be used to let off the load a bit by using various
+    statistical methods for testing. The Kernel Functions would soon be implemented in CUDA therefore this project work would prove to be the genesis of the understanding that the CUDA implementations are
+    correct and good to work with.
+mentors:
+  - Jim Pivarski (Princeton)
+  - Ianna Osborne (CERN)
```

Comment:
@SantamRC - please, change my affiliation from CERN to Princeton. Thanks!

Reply:
Updated . Thanks